src/interfaces.jl

```diff
@@ -150,7 +150,7 @@ function apply_convert!(P, attributes::Attributes, x::PlotSpec{S}) where S
     for (k, v) in pairs(kwargs)
         attributes[k] = v
     end
-    return (plottype(P, S), args)
+    return (S, args)
```

Comment:
@SimonDanisch, I'm trying your suggestion here (which fixes the `ECDFPlot` bug) but I was wondering: should this actually be `(plottype(S, P), args)`, so that the plot type suggested by the conversion pipeline, i.e. `S`, takes precedence, but if that is `Any`, it falls back to `P`?

Reply:
Ah... maybe that was the intention... Yeah, if `plottype(S, P)` does that, it sounds like a plan ;)